frontend/src/components/Course/Grading/GradingTest.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import { useState } from 'react';
+const GradingTest = (props: any) => {
+    const [inputCoefficientValue, setInputCoefficientValue] = useState('1');
+
+    const handleCoefficientValueChange = (e: any) => {
+        setInputCoefficientValue(e.target.value);
+        props.handleTestCoefficientChangeValue(e.target.value, props.test.id, props.test.exercise);
+    }
+
+    useEffect(() => {
+        if (props.test)
+            setInputCoefficientValue(props.test.coefficient || "1")
+    }, [props.test])
+
+    return (
+        <div className="flex items-center h-10">
+            <span className="bg-blue-50 border border-blue-100 rounded-l-lg text-gray-700 h-10 flex items-center px-3">
+                {props.test.name}
+            </span>
+            <input
+                type="number"
+                id="nota"
```

Comment:
Maybe it's better to use more explicit ids ? such as "notation", in this case

Reply:
Done